Automattic/vip-cli
# FORNO-556: Add media Import progress command

## Description

Adds `vip media-import status` command, similar to `vip site-import progress`.

## Steps to Test

Make sure you have a running local copy of GOOP, Parker, MMS.

1. Check out PR.
1. Run `npm run build`
1. Run `VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip-import-media-status.js @1000.production`
1. Verify the command is showing the status of the running import - although no import is really running there.
2. To simulate the import progressing, try connecting to the DB and changing the status manually to all statues until `COMPLETED` and see how the commands behaves.
3. Also try increasing/changing the number of processed files and total files and see if the CLI reflects the changes.
